ui/seek_bar.js

```diff
@@ -191,7 +192,11 @@ shaka.ui.SeekBar = class extends shaka.ui.RangeElement {
     }
   }
 
-  /** @return {boolean} */
+  /**
+   *  @return {boolean} 
```

Comment:
When using `@override`, you don't need to specify the other jsdocs unless they differ from the base/interface.

Reply:
Done